pom.xml

```diff
@@ -1,151 +1,209 @@
         </repository>
         <repository>
-            <id>Spigot</id>
-            <url>https://hub.spigotmc.org/nexus/content/groups/public/</url>
+            <!-- Floodgate -->
+            <id>opencollab-snapshot-repo</id>
+            <url>https://repo.opencollab.dev/maven-snapshots/</url>
+            <releases>
+                <enabled>false</enabled>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
         </repository>
         <repository>
-            <id>placeholderapi</id>
+            <!-- TARDIS -->
+            <id>tardis-repo</id>
+            <url>http://tardisjenkins.duckdns.org:8080/plugin/repository/everything/</url>
+        </repository>
+        <repository>
+            <!-- PlaceholderAPI -->
+            <id>placeholderapi-repo</id>
             <url>https://repo.extendedclip.com/content/repositories/placeholderapi/</url>
         </repository>
         <repository>
-            <id>papermc</id>
-            <url>https://papermc.io/repo/repository/maven-public/</url>
+            <!-- WorldEdit, WorldGuard -->
+            <id>sk89q-repo</id>
+            <url>https://maven.enginehub.org/repo/</url>
+        </repository>
+        <repository>
+            <!-- Various -->
+            <id>jitpack.io</id>
+            <url>https://jitpack.io</url>
+        </repository>
+        <repository>
+            <!-- Multiverse -->
+            <id>multiverse-repo</id>
+            <url>https://repo.onarandombox.com/content/repositories/multiverse-snapshots/</url>
+        </repository>
+        <repository>
+            <!-- BlockLocker -->
+            <id>codemc-repo</id>
+            <url>https://repo.codemc.org/repository/maven-public/</url>
+        </repository>
+        <repository>
+            <!-- LWCX (Lightweight Chests Extended) -->
+            <id>ender-zone-repo</id>
+            <url>https://ci.ender.zone/plugin/repository/everything/</url>
+        </repository>
+        <repository>
+            <!-- ProtocolLib -->
+            <id>dmulloy2-repo</id>
+            <url>https://repo.dmulloy2.net/repository/public/</url>
+        </repository>
+        <repository>
+            <!-- LibsDisguises -->
+            <id>md_5-public</id>
+            <url>https://repo.md-5.net/content/repositories/public/</url>
+        </repository>
+        <repository>
+            <!-- EssentialsX releases -->
+            <id>essentialsx-repo</id>
+            <url>https://repo.essentialsx.net/releases/</url>
+        </repository>
+        <repository>
+            <!-- EssentialsX snapshots -->
+            <id>essentialsx-snapshot-repo</id>
+            <url>https://repo.essentialsx.net/snapshots/</url>
+        </repository>
+        <repository>
+            <!-- RedProtect -->
+            <id>redprotect-repo</id>
+            <url>https://raw.githubusercontent.com/FabioZumbi12/RedProtect/mvn-repo/</url>
```

Comment:
400: Invalid request

Reply:
Trust me, it was on the GitHub for RedProtect, and works. I do not know why the link is broken, but it works. It is weird.